Resources/Prototypes/Radium/Entities/Mobs/Player/voxes.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  parent: MobVox
+  id: MobThiefVox
+  categories: [ HideSpawnMenu ]
+  name: Thief Vox
+  components:
+  - type: RandomHumanoidAppearance
+  - type: NpcFactionMember
+    factions:
+    - Syndicate
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)